app/controls/form/form.js

```diff
@@ -179,6 +179,12 @@ define([
                 'maxlength'     : null
             }, this.options, $el.data());
 
+            // Set the placeholder
+            var placeholder_text = $el.attr('placeholder');
```

Comment:
```data-placeholder``` is attached to the form items in the places that we use the form control. ```$el.data()``` will contain the placeholder value, what is the specific bug? Have you checked that this fixes it?